app/decorators/person_decorator.rb

```diff
@@ -85,6 +85,14 @@ def roles
     super.without_archived
   end
 
+  def current_roles_grouped
+    @current_roles_grouped ||= roles_grouped(scope: person.roles.without_future)
+  end
+
+  def future_roles_grouped
+    @future_roles_grouped ||= roles_grouped(scope: person.roles.future)
+  end
```

Comment:
Gelöschte Rollen sollen ebenfalls in einem eigenen Abschnitt aufgelistet werden.

Reply:
Gelöschte und archivierte Rollen werden jetzt zusammen unter "Inaktive Rollen" aufgelistet.